documentation/projects/proposals/rekognition_data/20240510-implementation_plan_machine_generated_tags_frontend.md

```diff
@@ -117,31 +113,33 @@ result pages of the Nuxt frontend.
-the accuracy data for the machine-generated providers.
-
-There are several options of displaying these properties.
-
-1. Do not display this data. The two tags sections will be identical.
-2. The generated tags will have a tooltip that displays the tag's provider and
-   its accuracy.
-3. Display generated tags in sections by provider so that we don't duplicate the
-   tag provider on each tag. Then, on each tag, display the accuracy.
-
-The first option is the simplest to implement, however, it fails to provide
-the fullest information to the users. The second option provides the accuracy
-and provider information to the user, however it is not very accessible.
-Different browsers and screen readers treat the tooltips differently. We have
-enough room on the page to display this data in full, using the third option:
-labelling all the tags generated by a single provider with a "by provider"
-heading, and adding the accuracy to the tag's inner content. This way, the
-user will see the provider and the accuracy of the tag without any additional
-actions. As a bonus, if a tag is generated by multiple providers, the user
-will see that different providers generate the same tag, and will be able to
-see the difference in accuracy between the providers.
+This project states that it is important to clearly distinguish the
+source-generated tags from the machine-generated tags.
+
+In addition to tag name, the machine-generated tags provide additional data
+points: their provider (currently, it will be "clarifai" and "rekognition") and
+their accuracy. This project aims to describe this data in a technical way on
+the "About tags" page, without displaying each individual tag's additional data,
+to prevent from overloading the users with too much information. Once we have
+more user data on how users relate to machine-generated tags, we will reconsider
+displaying this data on the single result page.
+
+Two other options were considered for displaying the machine-generated tags, but
+were discarded as they would be less user-friendly.
+
+1. Display the tag's provider and accuracy in a tooltip. This option would
+   provide the user all the information about the tag without cluttering the UI.
+   However, this option is not good for accessibility. Different browsers and
+   screen readers treat the tooltips differently.
+2. Display one section per each machine-generated tag provider, and label
+   sections with relevant heading, e.g. "Machine-Generated Tags: Rekognition".
+   In each section, list the tags in the order of their accuracy. Each tag would
+   display the tag name and the tag accuracy. This way, the user will see the
+   data without additional actions such as hovering. We have enough room on the
+   page to display this data in full. As a bonus, if a tag is generated by
+   multiple providers, the user will see that different providers generate the
+   same tag, and will be able to see the difference in accuracy between the
+   providers.
```

Comment:
Similar to point 1, do you mind adding a note here on why this was _not_ an ideal choice right now?